index.html

```diff
@@ -35,143 +40,173 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
+            <li class="dev">back end dev</li>
+            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+            <li class="dev">2015</li>
           </ul>
-        <form action="#">
-            <button class="btn-body" type="submit" name="submit">see project</button>
-        </form>
+          <p class="body-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+              <ul class="work-bottom work-bottom1">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <ul class="work-bottom2">
+                <li>html</li>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li class="js">javascript</li>
+              </ul>
+          <form action="#">
+              <button class="btn-body btn" type="submit" name="submit">see project</button>
+          </form>
+        </div>
     </section>
     <!-- Fourth section ends -->
     <!-- Body introduction section starts -->
     <section class="about-myself">
-      <h1 class="myself-header">about myself</h1>
-      <p class="myself-paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
-      <a class="connect" href="#">let's connect</a>
-      <nav class="icon-body">
-        <ul class="social-icons">
-          <li><a href="https://github.com/VictorChukwuebuka"><i class="fab fa-github"></i></a></li>
-          <li><a href="https://www.linkedin.com/in/chukwuebuka-ozoede-46616a219/"><i class="fab fa-linkedin-in"></i></a></li>
-          <li><a href=""><i class="fab fa-angellist"></i></a></li>
-          <li><a href=""><i class="fab fa-twitter"></i></a></li>
-          <li><a href=""><i class="fab fa-medium-m"></i></a></li>
-        </ul>
-      </nav> 
-      <form action="#">
-        <button class="btn-body" type="submit" name="submit" id="submit">get my resume</button>
-      </form>
-      <ul class="language">
-        <li class="dropdowns">languages</li>
-        <li><img src="./img/up-brace.svg" alt="dropdown-icon"></li>
-      </ul>    
-      <nav class="program-languages">
-        <ul class="script">
-          <li><img src="./img/javascript.svg" alt="js-logo"></li>
-          <li>JavaScript</li>
-        </ul>
-        <ul class="html">
-          <li><img src="./img/html-logo.svg" alt="html-logo"></li>
-          <li>html</li>
+      <div class="flex-item1 flex-item">
+        <h1 class="myself-header">about myself</h1>
+        <p class="myself-paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
```

Comment:
- [ ] Optional suggestions

- Work on the `letter-spacing` in your paragraph to get same we have in the figma.

- You can also change the `JavaScript HTML CSS ` seem to be broken.